src/relayFeeCalculator/chain-queries/arbitrum.ts

```diff
@@ -1,12 +1,16 @@
 import { QueryInterface } from "../relayFeeCalculator";
-import { BigNumberish } from "../../utils";
-import { BigNumber, providers } from "ethers";
+import {
+  BigNumberish,
+  createUnsignedFillRelayTransaction,
+  estimateTotalGasRequiredByUnsignedTransaction,
+} from "../../utils";
+import { providers } from "ethers";
 import { SymbolMapping } from "./ethereum";
 import { Coingecko } from "../../coingecko/Coingecko";
-import { ArbitrumSpokePool__factory, ArbitrumSpokePool } from "@across-protocol/contracts-v2";
+import { ArbitrumSpokePool__factory, SpokePool } from "@across-protocol/contracts-v2";
```

Comment:
nit: shouldn't these factories all be changed to be SpokePool__factory? I'm a bit surprised that the types are working (I didn't think that on-chain inheritance implies typechain type inheritance, but that's cool!). Anyway, it's probably a good idea to use the same factory type as the contract type you're using just to avoid any type issues in the future.